package.json

```
@@ -0,0 +1,32 @@
+{
+    "name": "dnd-beyond-dice-chrome-extension",
+    "version": "1.0.0",
+    "description": "A Chrome Extension that listens for dice rolls on D&D Beyond Character Sheets (e.g. [Remigius](https://www.dndbeyond.com/profile/kamnikite/characters/23934295)) and posts the results into a Discord server via a bot.",
+    "main": "content.js",
+    "scripts": {
+        "test": "echo \"Error: no test specified\" && exit 1"
+    },
+    "repository": {
+        "type": "git",
+        "url": "git+https://github.com/mikemfleming/dnd-beyond-dice-chrome-extension.git"
```

I think this should point to the original project, not to forks, but I'm unsure what this URL is used for.
```suggestion
        "url": "git+https://github.com/mazzola/dnd-beyond-dice-chrome-extension.git"
```

---

That's a good question... This is all output from initializing npm with `npm init -y` which probably assumes that the current git user is the maintainer. 

I think it should point to you, I'll set it up.